web_app/tests/test_positions.py

```diff
@@ -0,0 +1,446 @@
+                "deposit_data": {
+                    "token": "mock_token",
+                    "amount": "mock_amount",
+                    "multiplier": "mock_multiplier",
+                },
+            },
+        ),
+        (
+            "valid_wallet_id_3",
+            "SOL",
+            "1500",
+            3,
+            {
+                "contract_address": "mock_contract_address",
+                "position_id": "123",
+                "caller": "mock_caller",
+                "pool_price": 100,
+                "pool_key": {
+                    "token0": "mock_token0",
+                    "token1": "mock_token1",
+                    "fee": "mock_fee",
+                    "tick_spacing": "mock_tick_spacing",
+                    "extension": "mock_extension",
+                },
+                "deposit_data": {
+                    "token": "mock_token",
+                    "amount": "mock_amount",
+                    "multiplier": "mock_multiplier",
+                },
+            },
+        ),
+    ],
+)
+@pytest.mark.anyio
+async def test_create_position_success(
+    client: AsyncClient, wallet_id, token_symbol, amount, multiplier, expected_response
+) -> None:
+    """
+    Test for successfully creating a position with valid form data.
+
+    Args:
+        client (AsyncClient): The test client for the FastAPI application.
+
+    Returns:
+        None
+    """
+    mock_position = Mock()
+    mock_position.id = 123
+    mock_deposit_data = {
+        "caller": "mock_caller",
+        "pool_price": 100,
+        "pool_key": {
+            "token0": "mock_token0",
+            "token1": "mock_token1",
+            "fee": "mock_fee",
+            "tick_spacing": "mock_tick_spacing",
+            "extension": "mock_extension",
+        },
+        "deposit_data": {
+            "token": "mock_token",
+            "amount": "mock_amount",
+            "multiplier": "mock_multiplier",
+        },
+        "contract_address": "mock_contract_address",
+        "position_id": "123",
+    }
+
+    with (
+        patch(
+            "web_app.db.crud.PositionDBConnector.create_position"
+        ) as mock_create_position,
+        patch(
+            "web_app.contract_tools.mixins.deposit.DepositMixin.get_transaction_data"
+        ) as mock_get_transaction_data,
+        patch(
+            "web_app.db.crud.PositionDBConnector.get_contract_address_by_wallet_id"
+        ) as mock_get_contract_address,
+    ):
+        mock_create_position.return_value = mock_position
+        mock_get_transaction_data.return_value = mock_deposit_data
+        mock_get_contract_address.return_value = "mock_contract_address"
+        response = await client.post(
+        "/api/create-position-with-transaction-data",
+        data={
+        "wallet_id": wallet_id,
+        "token_symbol": token_symbol,
+        "amount": amount,
+        "multiplier": multiplier,
+            },
+            )   
+        assert response.ok
```

Comment:
Please, change it to `response.is_success` ,this object, doesn't have `ok` method, sorry